packages/frontend/src/components/header/DetailsHeader.tsx

```diff
@@ -69,6 +71,7 @@ export function DetailsHeader(props: HeaderProps) {
                 />
               )}
               {props.title}
+              {props.implementationHasChanged && <ImplementationUnderReview />}
```

Comment:
Didn't we decide to use baner?

Reply:
We didn't, but if you think it's better for consistency reasons, it's ok to have the usual banner in the header instead of the icon next to the name, just with the updated text